grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -147,48 +152,108 @@ def testFailure(self):
       self._RunQuery("UPDATE time SET day = -1;")
 
 
-class FakeOsqueryFlowTest(flow_test_lib.FlowTestsBaseclass):
+def _CreateDictTable(column_number, row_number) -> List[OrderedDict]:
+  def CellValue(col_number, row_number):
+    return (f"col-{col_number}", f"col-{col_number}, row-{row_number}")
+
+  return [
+    OrderedDict(
+      CellValue(col_number, row_number) for col_number in range(column_number)
+    ) for row_number in range(row_number)
+  ]
 
+
+class FakeOsqueryFlowTest(flow_test_lib.FlowTestsBaseclass):
   def setUp(self):
     super(FakeOsqueryFlowTest, self).setUp()
     self.client_id = self.SetupClient(0)
 
-  def _RunQuery(self, query):
+  def _NewTestFlowStartAndRun(self, query):
```

Comment:
Sorry I overlooked - I didn't rename `_RunQuery`, I separated some of its functionality in `_NewTestFlowStartAndRun`. But it's true that it wasn't obvious to me in the beginning that it starts a new flow. Does renaming it to `_RunQueryInNewFlow` sound good to you, or is it too wordy?

In general the scope of the two functions is not big, so I agree that maybe shorter names would be a better fit. I just prefer to be more explicit in the names, so that the code reads like a story. What do you think would be the best names for the two functions?